CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- `file:bits_per_sample`: now in `raster:bands.bits_per_sample`
+- `file:data_type`: now in `raster:bands.data_type`
+- `file:nodata`: now in `raster:bands.nodata`
+- `file:unit`: now in `raster:bands.unit`
+- `file:values`
```

Comment:
Looks alright, thanks, but we should also provide the alternative for file:values before removing it. (Unfortunately, my time is limited right now.)

Reply:
I think it's also ok to just leave it in for now, and remove when we have an alternative.